tests/pipeline/test_domain.py

```diff
@@ -312,17 +323,30 @@ def _test_equity_calendar_domain(self,
 
         assert_equal(actual, expected, check_names=False)
 
-    def test_equity_calendar_domain(self):
+    def test_built_in_equity_calendar_domain_defaults(self):
         # test the defaults
-        self._test_equity_calendar_domain(US_EQUITIES, datetime.time(8, 45))
-        self._test_equity_calendar_domain(CA_EQUITIES, datetime.time(8, 45))
-        self._test_equity_calendar_domain(GB_EQUITIES, datetime.time(7, 15))
+        expected_cutoff_times = {
+            BE_EQUITIES: datetime.time(8, 15),
+            CA_EQUITIES: datetime.time(8, 45),
+            CH_EQUITIES: datetime.time(8, 15),
+            DE_EQUITIES: datetime.time(8, 15),
+            FR_EQUITIES: datetime.time(8, 15),
+            GB_EQUITIES: datetime.time(7, 15),
+            JP_EQUITIES: datetime.time(8, 15),
+            NL_EQUITIES: datetime.time(8, 15),
+            PT_EQUITIES: datetime.time(7, 15),
+            US_EQUITIES: datetime.time(8, 45),
+        }
```

Comment:
Can we add an assertion here that the set of keys in this dict matches the set of entries in BUILT_IN_DOMAINS. I think the main thing we want from this test is to make sure that all the built-in domain definitions can correctly resolve their calendars names, so I want to make sure we're covering all of them.

Reply:
updated tests